pyuvdata/uvdata.py

```diff
@@ -910,6 +886,67 @@ def set_uvws_from_antenna_positions(self, allow_phasing=False,
             self.phase(phase_center_ra, phase_center_dec, phase_center_epoch,
                        phase_frame=output_phase_frame)
 
+    def reorder_pols(self, order='AIPS', desired_order=None, run_check=True,
```

Comment:
It doesn't look like keyword `desired_order` is used (or documented).

Reply:
oops, thanks for catching that. It's cruft.